x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/index.tsx

```diff
@@ -66,14 +66,16 @@ const ActionsComponent: React.FC<Props> = ({
       )}
       <EventsTd key="expand-event">
         <EventsTdContent textAlign="center" width={DEFAULT_ICON_BUTTON_WIDTH}>
-          <EuiButtonIcon
-            aria-label={expanded ? i18n.COLLAPSE : i18n.EXPAND}
-            data-test-subj="expand-event"
-            disabled={expanded}
-            iconType="arrowRight"
-            id={eventId}
-            onClick={onEventToggled}
-          />
+          <EuiToolTip data-test-subj="expand-event-tool-tip" content={i18n.EXPAND_EVENT}>
+            <EuiButtonIcon
+              aria-label={expanded ? i18n.COLLAPSE : i18n.EXPAND}
```

Comment:
I think now the `aria-label` should be always set to `Expand` as we don't allow to toggle it